src/devices/tuya.ts

```diff
@@ -6072,6 +6072,18 @@ const definitions: Definition[] = [
         toZigbee: [tzZosung.zosung_ir_code_to_send, tzZosung.zosung_learn_ir_code],
         exposes: [ez.learn_ir_code(), ez.learned_ir_code(), ez.ir_code_to_send()],
     },
+    {
+        fingerprint: [{modelID: 'TS1201', manufacturerName: '_TZ3290_rlkmy85q4pzoxobl'}],
```

Comment:
Since the code is exactly the same as `ZS06`, could you add it as a `tuya.whitelabel` there?

Reply:
Sorry I did not realize tuya.whitelabel before. Thanks for your reminder. Has added as instructed